lib/tenstorrent/event/event.c

```diff
@@ -19,7 +19,7 @@ uint32_t tt_event_wait(uint32_t events, k_timeout_t timeout)
 {
 	uint32_t ret;
 
-	ret = k_event_wait_safe(&tt_event, events, timeout);
+	ret = k_event_wait_safe(&tt_event, false, events, timeout);
```

Comment:
Does the `bool` argument come after the `events` argument?
https://github.com/zephyrproject-rtos/zephyr/blob/main/include/zephyr/kernel.h

Reply:
Fixed!